proxysql-admin

```diff
@@ -1964,9 +1971,9 @@ function parse_args() {
     SLAVEREAD_HOSTGROUP_ID=$READ_HOSTGROUP_ID
   fi
 
-  if [[ ! $WRITER_IS_READER =~ ^(always|never|ondemand)$ ]]; then
+  if [[ ! $WRITER_IS_READER =~ ^(0|1|2)$ ]]; then
     error "" "Invalid --writer-is-reader option: '$WRITER_IS_READER'"
```

Comment:
proxysql-admin -d is failing with following error.
```
ramesh@qaserver-05:/qa/8pxc/work/proxysql-admin-tool$ sudo ./proxysql-admin -d
ERROR : Invalid --writer-is-reader option: 'ondemand'
Please choose one of these values: 0(disabled), 1(always), or 2(only backup writer is reader)
ramesh@qaserver-05:/qa/8pxc/work/proxysql-admin-tool$
```
Variable `WRITER_IS_READER` is present in default configuration file proxysql-admin.cnf. We need to update the variable there too.
```
ramesh@qaserver-05:/qa/8pxc/work/proxysql-admin-tool$ sudo grep -B2 "WRITER_IS_READER" /etc/proxysql-admin.cnf

# Writer-is-reader configuration
export WRITER_IS_READER="ondemand"
ramesh@qaserver-05:/qa/8pxc/work/proxysql-admin-tool$
``` 
bash -x outptut
```
+ SLAVEREAD_HOSTGROUP_ID=21
+ [[ ! ondemand =~ ^(0|1|2)$ ]]
+ error '' 'Invalid --writer-is-reader option: '\''ondemand'\'''
+ local lineno=
+ shift
+ [[ -n '' ]]
+ printf 'ERROR : Invalid --writer-is-reader option: '\''ondemand'\''\n'
ERROR : Invalid --writer-is-reader option: 'ondemand'
+ echo 'Please choose one of these values: 0(disabled), 1(always), or 2(only backup writer is reader)'
Please choose one of these values: 0(disabled), 1(always), or 2(only backup writer is reader)
+ exit 1
ramesh@qaserver-05:/qa/8pxc/work/proxysql-admin-tool$
```

Reply:
@rameshvs02, yes, WRITER_IS_READER option in ProxySQL 2.0 is not compatible. It is expected